packages/cli/src/lib/config.ts

```diff
@@ -14,13 +14,14 @@
  *  limitations under the License.
  */
 
-import {existsSync} from 'fs';
+
 import {join} from 'path';
 import {homedir} from 'os';
 import {Config} from '@bubblewrap/core';
 import * as inquirer from 'inquirer';
+import {existsSync, promises} from 'fs';
 
-const DEFAULT_CONFIG_PATH = join(homedir(), '/.llama-pack/llama-pack-config.json');
+const DEFAULT_CONFIG_PATH = join(homedir(), '/.bubblewrap-config/bubblewrap-config.json');
```

Comment:
Do you mind making 4 variables:

`DEFAULT_CONFIG_FOLDER` - ~/.bubblewrap-config
`DEFAULT_CONFIG_NAME` - bubblewrap-config.json
`LEGACY_CONFIG_FOLDER` - ~/.llama-pack
`LEGACY_CONFIG_NAME` - llama-pack-config.json

And then using them below?